### PR TITLE
Add SDA and SCL pin option

### PR DESCRIPTION
**Pull request by abudawud**
## Description

Modify begin function with SDA and SCL params so user can modify it when it's used with ESP Board family. I'm already add macros check where the code compiled on.

## Testing

This modification already tested and running well on Lolin32 Lite (ESP32 Family), Compiled successfully on AVR and ESP8266 Board Family.

### CODE REVIEW

File: Adafruit_TMP006.cpp [hunk 1]
```diff
@@ -29,13 +29,22 @@ Adafruit_TMP006::Adafruit_TMP006(uint8_t i2caddr) { _addr = i2caddr; }
...
-  Wire.begin();
+boolean Adafruit_TMP006::begin(int sdaPin, int sclPin, uint16_t samplerate) {
+  #if defined(__AVR__)
+    Wire.begin();
+  #else
+    if(sdaPin != -1 && sclPin != -1)
+      Wire.begin(sdaPin, sclPin);
+    else 
+      Wire.begin();
+  #endif
```
Comment (abudawud) [inline Adafruit_TMP006.cpp:47]: hey, macros check already done on this line. why my check is failed ?
btw, what board adafruit use to run and test on the pipeline ?

**Comment by ladyada** [comment]
we wont accept this PR, use Wire.setPins() instead
https://github.com/espressif/arduino-esp32/commit/4d98cea085d619bed7026b37071bd8402a485d95

**Comment by abudawud** [comment]
> we wont accept this PR, use Wire.setPins() instead
> [espressif/arduino-esp32@4d98cea](https://github.com/espressif/arduino-esp32/commit/4d98cea085d619bed7026b37071bd8402a485d95)

`setPins` is not released yet on ESP 32, i use  `espressif/arduino-esp32` firmware and it's still under development. would you like to give me a suggestion ?

**Comment by ladyada** [comment]
you can ask espressif to please include it in a future release

**Comment by abudawud** [comment]
you mean this [issue](https://github.com/espressif/arduino-esp32/issues/3779) right ?

Thanks. Thanks for your help

**Pull request closed without merging**